### DISCUSSION TITLE
I can help you to translate this plugin into Vietnamese for FREE

### DISCUSSION THREAD

**Discussion by ghost** [fuyutsuki/Texter#105] [Ideas]
I can help you because there are lots of server that use your plugin and some of them don't know English

**Comment by fuyutsuki** [comment]
First, please translate the currently translated [`en_us.ini`](../blob/release/v4.0.0/resources/language/en_us.ini) and create a Vietnamese translation file (`vi_vn.ini`).
Then, place it in `resources/languages` and create a PR.
Optionally, it would be great if you could also translate the `README.md` and place it as `.github/readme/vi_vn.md`.

I appreciate your help!
